recipes-omnect/images/omnect-os-update-image.bb

```diff
@@ -36,18 +42,27 @@ do_bootloader_package:rpi() {
     tar -czvf boot-partition-update.tar.gz -C ${DEPLOY_DIR_IMAGE}/boot-partition .
 }
 
-do_bootloader_package() {
+do_bootloader_package:omnect_uboot() {
```

Comment:
imho we should  here use :phytec-imx8mm

Reply:
nope, that would not work for raspberry devices, omnect_uboot is the right one imo.

Reply:
done